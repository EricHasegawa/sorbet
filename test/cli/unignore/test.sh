#!/bin/bash

main/sorbet --silence-dev-message --ignore usually_ignored/ --unignore /subfolder --ignore foo.rb --unignore foo.rb --dir test/cli/unignore 2>&1
