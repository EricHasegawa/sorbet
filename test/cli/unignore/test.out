subfolder/usually_ignored/dont_ignore.rb:4: Expected `Integer` but found `String("subfolder/usually_ignored/dont_ignore.rb")` for argument `arg0` https://srb.help/7002
     4 |1 + "subfolder/usually_ignored/dont_ignore.rb"
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  Expected `Integer` for argument `arg0` of method `Integer#+`:
    https://github.com/sorbet/sorbet/tree/master/rbi/core/integer.rbi#L148:
     148 |        arg0: Integer,
                  ^^^^
  Got `String("subfolder/usually_ignored/dont_ignore.rb")` originating from:
    subfolder/usually_ignored/dont_ignore.rb:4:
     4 |1 + "subfolder/usually_ignored/dont_ignore.rb"
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

foo.rb:4: Expected `Integer` but found `String("foo.rb")` for argument `arg0` https://srb.help/7002
     4 |1 + "foo.rb"
            ^^^^^^^^
  Expected `Integer` for argument `arg0` of method `Integer#+`:
    https://github.com/sorbet/sorbet/tree/master/rbi/core/integer.rbi#L148:
     148 |        arg0: Integer,
                  ^^^^
  Got `String("foo.rb")` originating from:
    foo.rb:4:
     4 |1 + "foo.rb"
            ^^^^^^^^
Errors: 2
